src/Confluent.SchemaRegistry/Utils.cs

```diff
@@ -39,19 +39,6 @@ public static bool DictEquals(IDictionary<string, string> a, IDictionary<string,
             }
             return true;
         }
-        
-        public static bool DictEquals(IDictionary<string, ISet<string>> a, IDictionary<string, ISet<string>> b)
```

Comment:
I’ve removed this since it’s no longer used. Technically, this could be considered a breaking change to the public API, though I’m not sure utility methods like this should be part of the public surface in the first place.

In the future, we could consider keeping such methods internal and use `System.Runtime.CompilerServices.InternalsVisibleTo` if we need to share them with related packages.